examples/sourceview/dune

```diff
@@ -0,0 +1,5 @@
+(executables
+ (names test2 sourceview2 example2)
```

Comment:
maybe you should update the name of the `sourceview2` example?

Reply:
do you mean rename it `sourceview3`? i should probably rename all three examples, then (the readme says the examples ending with a `2` are the lablgtk3 ones, so i didn't change it, but i probably should have)

Reply:
For example to `sourceview3`, I guess the name is misleading otherwise right? IMHO feel free to rename them so their names are more illustrative.

Reply:
done